includes/definitions/discovery/ciena-sds.yaml

```diff
@@ -8,26 +8,90 @@ modules:
                 num_oid: '.1.3.6.1.4.1.1271.2.1.5.1.2.1.4.5.1.4.{{ $index }}'
                 descr: 'Slot {{ $subindex1 }} CPU'
                 snmp_flags: '-OQUsbe'
+            -
+                oid: wwpLeosSystemCpuUtilization
+                value: wwpLeosSystemCpuUtilizationLast5Seconds
+                num_oid: '.1.3.6.1.4.1.6141.2.60.12.1.11.1.{{ $index }}'
+                descr: 'CPU Utilization 5'
+                index: wwpLeosSystemCpuUtilizationLast5Seconds.{{ $index }}
```

Comment:
This index is too long for the database :D
Just shorten it up.

Reply:
Ok, I thought the issue was with the test data. No wonder I was lost.

I guess I just have to change it to the "num_oid" field?